packages/contracts/src/dollar/interfaces/IBancorFormula.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
```

Comment:
Shouldn't all of these be updated to the latest Solidity version?

Reply:
It uses the latest version available in the 0.8 range